motoman_driver/MotoPlus/SimpleMessage.h

```diff
@@ -298,33 +312,33 @@ typedef struct _SmBodyMotoWriteIOBit SmBodyMotoWriteIOBit;
 
 struct _SmBodyMotoWriteIOBitReply
 {
-	UINT32 resultCode;
+	IoResultCodes resultCode;
 } __attribute__((__packed__));
 typedef struct _SmBodyMotoWriteIOBitReply SmBodyMotoWriteIOBitReply;
 
 struct _SmBodyMotoReadIOGroup
 {
-	UINT32 ioAddress;
+	UINT32 ioAddress; //Group address. Example: '1001' will read OG#1 (10010 - 10017)

```

Comment:
```suggestion
	UINT32 ioAddress; //Group address. Example: '1001' will read OG#1 (IOs / bits 10010 to 10017)
```
The `-` confused me at first, I thought there was a subtraction there.

Reply:
And the same below.

Reply:
This has been updated.